app/controllers/reviews_controller.rb

```diff
@@ -9,14 +9,17 @@ def create
     @card = Card.find(review_params[:card_id])
 
     case @card.check_answer(review_params[:answer])
-      when :correct_answer
-        redirect_to root_path, notice: 'Верно!'
-      when :wrong_answer
-        redirect_to root_path, alert: 'Ваш ответ не правильный'
-      when :wrong_answers_streak
-        flash[:remind] = 'Вы ввели три раза не правильно!
-                            Сделующая дата пересмотра карточки через 12 часов.'
-        redirect_to root_path
+    when :correct_answer
+      redirect_to root_path, notice: 'Верно!'
+    when :typo_in_word
+      flash.now[:remind] = "Возможно вы ввиду слово #{@card.original_text}."
+      render :show
+    when :wrong_answer
+      redirect_to root_path, alert: 'Ваш ответ не правильный'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.